test/local-content/collections/ansible_collections/testns/testcoll/plugins/modules/test_module_2.py

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/python
+# pylint: disable=cyclic-import
```

Comment:
This should have a note pointing to the upstream issue as the motivation for adding an ignore

Reply:
@clickthisnick Can you please address that? If not I will close it because there are not so many benefits from doing this location swap.